mock-identity-system/src/main/java/io/mosip/mock/identity/system/validator/RequestTimeValidator.java

```diff
@@ -0,0 +1,39 @@
+package io.mosip.mock.identity.system.validator;
+
+import static io.mosip.mock.identity.system.util.Constants.UTC_DATETIME_PATTERN;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RequestTimeValidator implements ConstraintValidator<RequestTime, String> {
+
+	@Value("${mosip.mock.authdata.reqtime.maxlimit:-2}")
+    private int maxMinutes;
+
+	@Value("${mosip..mock.authdata.reqtime.minlimit:2}")
+    private int minMinutes;
```

Comment:
Can we take the above 2 configs as a single config called "esignet.mock.identity-system.allowed-request-time-variation-seconds" and do both + and - case.